data/water/wetlands/index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: Wetlands
+title: Wetlands and Riparian Areas
```

Comment:
**chore**: let's remove the title and use the the abstract title that is generated below.
```suggestion
```